File: backend/core/src/main/java/site/timecapsulearchive/core/domain/capsule/data/secret_c/SecretCapsuleDetail.java
package site.timecapsulearchive.core.domain.capsule.data.secret_c;

import java.time.ZonedDateTime;
import site.timecapsulearchive.core.domain.capsule.entity.CapsuleType;

public record SecretCapsuleDetail(
    Long capsuleId,
    String capsuleSkinUrl,
    ZonedDateTime dueDate,
    String nickname,
    String profileUrl,
    ZonedDateTime createdAt,
    String address,
    String title,
    String content,
    Boolean isOpened,
    CapsuleType capsuleType
) {

}

File: frontend/ARchive/app/src/main/java/com/droidblossom/archive/domain/model/secret/SecretCapsuleDetail.kt
package com.droidblossom.archive.domain.model.secret

// 모든 캡슐 디테일로 통합될 예정 maybe
data class SecretCapsuleDetail(
    val address: String,
    val capsuleSkinUrl: String,
    val content: String,
    val createdDate: String,
    val dueDate: String?,
    val isOpened: Boolean,
    val imageUrls: List<String>?,
    val videoUrls : List<String>?,
    val nickname: String,
    val title: String
){
    constructor() : this("","","","",null,false, listOf(), listOf(),"","")
}
